src/test/java/io/kokuwa/maven/helm/UploadMojoTest.java

```diff
@@ -315,4 +369,78 @@ private void assertUpload(UploadMojo mojo, RequestMethod method, String path, St
 		assertEquals("application/gzip", request.getHeader(HttpHeaders.CONTENT_TYPE), "content-type");
 		assertEquals(authorization, request.getHeader(HttpHeaders.AUTHORIZATION), "authorization");
 	}
+
+	private void assertUploadVerifySuccess(UploadMojo mojo, RequestMethod method, String path) {
+		RequestPatternBuilder requestPattern;
+		requestPattern = RequestPatternBuilder.allRequests();
+
+		mock.stubFor(WireMock.put(WireMock.urlMatching(".*"))
+				.willReturn(WireMock.ok()
+						.withStatus(200)));
+		mock.stubFor(WireMock.get(WireMock.urlMatching(".*/index\\.yaml$"))
+				.willReturn(WireMock.ok()
+						.withStatus(200)
+						.withHeader("Content-Type", "application/yaml")
+						.withBody(getIndexYamlBody())));
+		mock.stubFor(WireMock.get(WireMock.urlMatching(".*\\.tgz$"))
+				.willReturn(WireMock.ok()
+						.withStatus(200)
+						.withHeader("Content-Type", "application/gzip")
+						.withBodyFile("app-0.1.0.tgz")));
+
```

Comment:
when helm chart show is run, it expects two returns from the repo: the index.yaml and the tar zipped charts. this is mocking that response to emulate the response of running helm chart show